### PR TITLE
Change the base image to resin/amd64-systemd:jessie

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
-FROM debian:jessie
+FROM resin/amd64-systemd:latest
```
Comment (petrosagg) [inline Dockerfile]: I think it makes sense to start `FROM resin/amd64-node:4.0`

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
 
-ENV DEBIAN_FRONTEND noninteractive
 ENV TERM xterm
 
-COPY src/01_nodoc /etc/dpkg/dpkg.cfg.d/
-COPY src/01_buildconfig /etc/apt/apt.conf.d/
-
 RUN apt-get update \
 	&& apt-get dist-upgrade \
```
Comment (petrosagg) [inline Dockerfile]: We don't need dist-upgrade anymore

File: Dockerfile [hunk 3]
```diff
@@ -55,26 +51,16 @@ WORKDIR /usr/src/app
...
-	sys-kernel-debug.mount \
-
-	display-manager.service \
-	getty@.service \
-	systemd-logind.service \
-	systemd-remount-fs.service \
-
-	getty.target \
-	graphical.target
+	sys-kernel-debug.mount
```
Comment (petrosagg) [inline Dockerfile]: Is this mark necessary? I think the container will work fine without this

File: Dockerfile [hunk 4]
```diff
@@ -55,26 +51,16 @@ WORKDIR /usr/src/app
 
 # systemd configuration
 
-ENV container lxc
-
 # We never want these to run in a container
 RUN systemctl mask \
-	dev-hugepages.mount \
 	dev-mqueue.mount \
```
Comment (petrosagg) [inline Dockerfile]: Is this mask necessary?

File: Dockerfile (hunk 1 above)
Comment (nghiant2710) [inline Dockerfile]: resin-base is using Node v4.2.6. I will need to update amd64-node base image first then we can change to that base image later

File: Dockerfile (hunk 2 above)
Comment (nghiant2710) [inline Dockerfile]: updated!

**Comment by Page-** [comment]
@petrosagg how does this interact with #1 for unprivileged running? Can those changes be added to the other images as well?

**Pull request closed without merging**